llvm/lib/Transforms/Coroutines/CoroFrame.cpp

```diff
@@ -2880,6 +2922,28 @@ void coro::salvageDebugInfo(
     Expr = DIExpression::prepend(Expr, DIExpression::DerefBefore);
   }
 
+  return {{Storage, Expr}};
+}
+
+void coro::salvageDebugInfo(
+    SmallDenseMap<Argument *, AllocaInst *, 4> &ArgToAllocaMap,
+    DbgVariableIntrinsic *DVI, bool OptimizeFrame, bool UseEntryValue) {
```

Comment:
this function assumes `DVI` is non null, let's express that at the API level by making it a reference